test/test_gateway_sockets.py

```diff
@@ -42,6 +55,132 @@ class GatewaySocketsTest(aiounittest.AsyncTestCase):
         response = await send_ws_message({"type": "socket-ping", "data": {}})
         self.assertEqual(response, {"alive": "True"})
 
+    async def test_fl_process(self):
```

Comment:
@cereallarceny , @Prtfw , @hericlesme  Please, take a look at this method.
This unit test method shows how to use the WebSocket interface to host a new training process ( sending serialized models, cycles and protocols), perform a simple authentication and request to join in a new FL cycle

Reply:
LGTM!  I will say that technically the `host_federated_training` method exists on PySyft, not PyGrid. But for testing purposes... that's okay. :)